theano/gpuarray/subtensor.py

```diff
@@ -1106,15 +1106,19 @@ def gpu_kernels(self, node, nodename):
   old = *base;
   do {
     assumed = old;
-    sum = __float2half_rn(
+    ga_half old_perm;
+    __HALF_TO_US(old_perm)  = __byte_perm(old, 0,
```

Comment:
According to @abergeron comment in your old PR ( here: https://github.com/Theano/Theano/pull/6283/files#r132486589 ), those definitions should be moved to libgpuarray in the future. So, maybe the `theano.gpuarray.subtensor` module should not be modified in this PR for the moment ?